spartan/aztec-network/values.yaml

```diff
@@ -29,6 +29,11 @@ telemetry:
   useGcloudLogging: true
   excludeMetrics: ""
 
+snapshots:
+  uploadLocation: "gs://aztec-testnet/snapshots/"
+  syncUrl: "https://storage.googleapis.com/aztec-testnet/snapshots/"
```

Comment:
IMHO I think these should be undefined in the base values file, but happy to leave it as is